ansible/playbooks/roles/encoder-network/files/nftables.conf

```
@@ -0,0 +1,15 @@
+table inet filter {
```

Let's keep some of the defaults from the Debian included config here.

```suggestion
#!/usr/sbin/nft -f
{{ ansible_managed | comment }}

flush ruleset

table inet filter {
```